### ISSUE TITLE
get_config function

### ISSUE THREAD

**Issue by jyan26** [Bihaqo/t3f#212]
In t3f nn.py, I was wondering if we could add get_config function to KerasDense class so that we could save the trained model? Right now, without get_config, I got this error message when trying to save the model or using callbacks in model.fit: "NotImplementedError: Layers with arguments in `__init__` must override `get_config`." (I'm using TF 2.0.3.) So maybe we could add something like (I'm not sure if the following is right):
```
def get_config(self):
  config = super(KerasDense,self).get_config()
  config.update({'input_dims': self.input_dims,
                 'output_dims': self.output_dims,
                 'tt_rank': self.tt_rank,
                 'activation': self.activation,
                 'use_bias': self.use_bias,
                 'kernel_initializer':self.kernel_initializer,
                 'bias_initializer': self.bias_initializer})
  return config
```

**Comment by Bihaqo** [comment]
Great suggestion, thank you! Are you ok with me copy pasting this into the
library? Or feel free to do it yourself and send a pull request if you want.

**Comment by jyan26** [comment]
Yes, sure, please feel free to copy and paste this code. I didn't have a chance to test this code though, so please test it along with other codes before adding it into the library. Thanks!
